presto-hive/pom.xml

```diff
@@ -332,8 +332,6 @@
                             <excludes>
                                 <!-- Exclude tests against Glue from build -->
                                 <exclude>**/TestHiveGlueMetastore.java</exclude>
-                                <!-- Parquet tests take a long time so only run them in the CI server -->
-                                <exclude>**/TestFullParquetReader.java</exclude>
```

Comment:
I looked at this earlier while doing the Parquet upgrade and it indeed takes a very long time, even on my laptop (newish Mac with 6 cores). Probably need to leave this disabled for now and file a separate issue to figure out how to address it.

Reply:
I think the ORC tests have the same problem (no output)